migration-overview.md

```diff
@@ -6,63 +6,61 @@ aliases: ['/docs/dev/migration-overview/']
 
 # Migration Overview
 
-This document describes how to migrate data to TiDB, including migrating data from MySQL and from CSV/SQL files.
+This document describes how to migrate data to TiDB, which provides the following data migration features:
 
-## Migrate from Aurora to TiDB
+- Full data migration: use `TiDB Lightning` to import data(CSV/Aurora Snapshot/mydumper sql) to TiDB cluster; to better match with full migration from MySQL/MariaDB database, TiDB also provides a data export tool, `dumpling`, which supports exporting full data as CSV/mydumper sql format;
 
-In a cloud environment, you can directly migrate full data to TiDB by exporting snapshot from Aurora. For details, see [Migrate from Amazon Aurora MySQL Using TiDB Lightning](/migrate-from-aurora-using-lightning.md).
+  - Fast-initializing TiDB clusters: `TiDB Lightning` also provides the fast import feature, which can achieve the effect of fast initialization of specified tables of TiDB clusters. Before using this feature, you need to understand that the fast import period has a great impact on TiDB clusters, and the clusters are not available;
```

Comment:
I'm not sure if it is proper to mention "Fast-initializing TiDB clusters" in this document, because it seems that "initializing TiDB clusters" belongs to the installation scenario, not the migration scenario.

Reply:
Is it the first import after installing the cluster, or I call it offline fast import, is it more suitable?